record:
  ros__parameters:
    output_dir: '/home/zainir17/ta_ws/record' # Directory to save the recordings
    lidar_list:
      - '/velodynekanan/velodyne_points'
      - '/lidartengah/lidar_points'
      - '/velodynekiri/velodyne_points'
    camera_list:
      - '/camerakiri/image_raw'
      - '/cameratengah/image_raw'
      - '/camerakanan/image_raw'
    # navsat_topic: '/fix' # GPS topic
    samp_time: 0.2 #seconds
